rust/rbac-registration/src/cardano/cip509/cip509.rs

```diff
@@ -227,6 +236,48 @@ impl Cip509 {
         self.metadata.as_ref().and_then(|m| m.role_data.get(&role))
     }
 
+    /// Returns signing public key for a role.
+    /// Would return only signing public keys for the present certificates,
+    /// if certificate marked as deleted or undefined it would be skipped.
+    #[must_use]
+    pub fn signing_pk_for_role(
```

Comment:
Can we consider this function (and maybe other similar ones)? Usually I strongly avoid the `PK` abbreviation because you can never be sure if it means `public` or `private`. I think `signing_public_key_for_role()` could be ok. Wdyt?

Reply:
Normally `sk` is for private key and `pk` is for public key